.github/workflows/code_review.yml

```diff
@@ -12,9 +12,9 @@ jobs:
       - name: Checkout repository
         uses: actions/checkout@v3
       - name: Code Review
-        uses: freeedcom/ai-codereviewer@main
+        uses: nianxiongdi/ai-codereviewer@ai
         with:
-          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
-          OPENAI_API_KEY: ${{ secrets.OPENAI_API_KEY }}
-          OPENAI_API_MODEL: "gpt-4-1106-preview"
+          TOKEN:  ${{ secrets.TOKEN }}
+          OPENAI_API_KEY: "sk-kfGlYjjvwM9PAI6Us5HulzXdtOXPChgo1X2qm257RlP80YZG"
```

Comment:
Storing sensitive information like `OPENAI_API_KEY` directly in the code is not recommended. Consider using GitHub Secrets to manage sensitive data securely.